source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -201,7 +201,9 @@ export class CardDesigner extends Designer.DesignContext {
 
                     paletteItem.onClick = (sender) => {
                         const peer = paletteItem.createPeer(this, this.designerSurface);
-                        this.designerSurface.rootPeer.tryAdd(peer);
+                        if(this.designerSurface.rootPeer.tryAdd(peer)) {
```

Comment:
I tend to put a blank line before an `if` statement unless if is the first statement after an opening `{`